integration_tests/integration/link/requestLink.spec.ts

```diff
@@ -9,14 +9,6 @@ context('Request Link Page', () => {
     cy.task('stubRequestLink')
   })
 
-  it('should render request link page to unauthenticated user and reset cookie', () => {
```

Comment:
Removed this test because we will no longer store the auth token in a cookie (future dev work - at the moment we dont store it anywhere)
The new approach will be to store the auth token in the session, but this test is harder to write based on persisting/clearing stuff in the session as I'm not sure cypress has access to the server side session does it?

Reply:
No - you might have to save that integration test for the end and do it via the sign out link.

Reply:
'No' as in 'no, dont remove the test'? or No as in 'no, cypress cant see the session so we'll have to test this differently via the sign out like at the end' ?

Reply:
```No as in 'no, cypress cant see the session so we'll have to test this differently via the sign out like at the end' ```